android/src/main/java/com/mrousavy/camera/CameraViewModule.kt

```diff
@@ -75,7 +81,14 @@ class CameraViewModule(reactContext: ReactApplicationContext) : ReactContextBase
     return TAG
   }
 
-  private fun findCameraView(id: Int): CameraView = reactApplicationContext.currentActivity?.findViewById(id) ?: throw ViewNotFoundError(id)
+  private fun findCameraView(viewId: Int): CameraView {
+    Log.d(TAG, "Finding view $viewId...")
+    val mContext = WeakReference(reactApplicationContext)
+    val ctx = mContext?.get()
```

Comment:
Hey - I think this is unnecessary. We should just be able to pass in `reactApplicationContext` to the function below without having to create a WeakReference and access it immediately afterwards.